_sass/_base.scss

```diff
@@ -1,19 +1,26 @@
 html {
-  background-color: lighten($border-color, 15%);
+  &.light {
+    background-color: #e1e1e1;
+  }
+  &.dark {
+    background-color: #626262;
+  }
```

Comment:
TODO: Add a comment explaining why this is done and `@include theme()` can't be used here.